src/components/NewGalaxyModal.js

```diff
@@ -88,6 +132,11 @@ export const NewGalaxyModal = _.flow(
             div({ style: { lineHeight: 1.5 } }, [
               div(['Please delete the cloud environment when finished; it will']),
               div(['continue to ', span({ style: { fontWeight: 600 } }, ['incur charges ']), 'if it keeps running.'])
+            ]),
+            div({ style: { ...styles.headerText, marginTop: '0.5rem' } }, ['Pause and auto-pause']),
+            div({ style: { lineHeight: 1.5 } }, [
+              div(['You can pause anything during the compute, but it will auto-pause when']),
```

Comment:
"anything" -> "at any time"? This message might benefit from a comms pass for clarity/grammar.